homeassistant/components/stream/worker.py

```diff
@@ -138,19 +147,11 @@ def __init__(
     ) -> None:
         """Initialize StreamMuxer."""
         self._hass = hass
-        self._segment_start_dts: int = cast(int, None)
-        self._memory_file: BytesIO = cast(BytesIO, None)
-        self._av_output: av.container.OutputContainer = None
-        self._input_video_stream: av.video.VideoStream = video_stream
-        self._input_audio_stream: av.audio.AudioStream | None = audio_stream
+        self._input_video_stream = video_stream
+        self._input_audio_stream = audio_stream
         self._audio_bsf = audio_bsf
-        self._audio_bsf_context: av.BitStreamFilterContext = None
-        self._output_video_stream: av.video.VideoStream = None
-        self._output_audio_stream: av.audio.AudioStream | None = None
+        self._audio_bsf_context: av.BitStreamFilterContext | None = None
```

Comment:
`self._audio_bsf_context` is set to `None` and the code already checks if it's not `None` when it should be used.